src/portable/raspberrypi/rp2040/dcd_rp2040.c

```diff
@@ -48,8 +48,8 @@
 // Init these in dcd_init
 static uint8_t *next_buffer_ptr;
 
-// Endpoints 0-15, direction 0 for out and 1 for in.
-static struct hw_endpoint hw_endpoints[16][2] = {0};
+// USB_MAX_ENDPOINTS Endpoints, direction 0 for out and 1 for in.
+static struct hw_endpoint hw_endpoints[USB_MAX_ENDPOINTS][2] = {0};
```

Comment:
16 pairs of endpoints are lots, 8 is more common. not really need changes though :smile: